pages/dashboard.js

```diff
@@ -92,187 +189,36 @@ export default function Dashboard(props) {
         myBenefitsAndServices={t.myBenefitsAndServices}
       />
       <div className="mb-8">
-        {cppLoaded ? null : 'Loading CPP User Benefit Data...'}
-        {cppError ?? null}
```

Comment:
I think it will be helpful to keep the unstyled loading and error text for the time being.